template-processors/applier/bin/processTemplates.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+
+cd $CLONED_TEMPLATE_GIT_DIR/
+ansible-galaxy install -r requirements.yml -p galaxy
+ansible-playbook -i .applier/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml
```

Comment:
Why do you have a yaml file hardcoded here?

Reply:
so, this is kind of how OpenShift Applier works.. we treat it as if its a CLI, not as if its ansible. It runs as a standard playbook/role that you customize the behavior of by way of the inventory, which is what gets pulled down from Git. We do this because some day it actually will be a non-ansible based CLI tool.